package.json

```diff
@@ -30,6 +30,7 @@
     "start-photon": "node res/photon/server",
     "test": "cross-env LC_ALL=C NODE_ENV=test jest",
     "test-all": "run-p --max-parallel 4 flow license-check lint test test-alex test-lockfile",
+    "test-all:ci": "run-p --max-parallel 4 license-check test-alex lint flow:ci test",
```

Comment:
Just wondering if the order is special? Otherwise I'd try to have it in the same order as the one on top, as this makes it easier to double-check. Also maybe add test-lockfile which is pretty fast anyway?

Reply:
The order is the same as the previous order in the appveyor.yml file. But I can make it same with the test-all. Will also add the test-lockfile then.